### DISCUSSION TITLE
Gollama tool for managing Ollama models

### DISCUSSION THREAD

**Discussion by AlloriMD** [ericmjl/llamabot#135] [General]
Here is a tool that may be of interest to those using Ollama with llamabot -- [**Gollama**](https://github.com/sammcj/gollama) is a command-line tool that can manage the LLM models that you have downloaded using Ollama. It's pretty helpful, since Ollama itself is pretty meager on the user interface. Gollama is run at the command line, but it is more like a TUI (text user interface) than simple CLI, and it has a nice design. An added feature of Gollama is the ability to share Ollama LLMs with LMStudio (so you don't have to download multiple copies of the same models, wasting disk space).

I figure a tip like this might be useful for the llamabot documentation.

**Comment by ericmjl** [comment]
That's a great find, @AlloriMD! Thank you so much for posting!
